bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/VBF_RS_bulk_M1500_W01pc/VBF_RS_bulk_M1500_W01pc_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+set group_subprocesses Auto
+define p = g u c d s u~ c~ d~ s~
```

Comment:
we suggest to remove the proton and jet definitions as they are handled automatically